### ISSUE TITLE
Invalid force and potential computation

### ISSUE THREAD

**Issue by Chelsea486MHz** [Garuda1/senpai#7]
So here's me realising for **months** I've been rendering simulations using invalid computations. Behold the extent of my misery:

- The Lennard-Jones force (`force_lennardjones`)
- The Lennard-Jones potential (`potential_lennardjones`)
- The bond angle potential (`potential_angle`)
- The electrostatic potential (`potential_electrostatic`)

All provide incorrect results, with consequences ranging from invalid potential reduction to dead on system explosion.

I'll now have a break, have a beer, and proceeed to f@#! kill myself

**Comment by Chelsea486MHz** [comment]
The following functions are now fixed:

- The Lennard-Jones force (`force_lennardjones`)
- The Lennard-Jones potential (`potential_lennardjones`)

Further work will have to decide if the results of `potential_electrostatic` are satisfactory or not.

**Comment by Chelsea486MHz** [comment]
The following functions have been identified as faulty:

- `force_lennardjones`
- `potential_lennardjones`
- `potential_angle`

And disabled until a fix is developped.

**Comment by Chelsea486MHz** [comment]
The following functions are now fixed:

- The Lennard-Jones force (`force_lennardjones`)
- The Lennard-Jones potential (`potential_lennardjones`)

**Comment by Chelsea486MHz** [comment]
Wooooohooooooo

Okay so `potential_angle` and `force_angle` are fixed.

The issue was lying in the fact that I was trying to relate forces to angular displacements, while I in fact had to relate torques. Which makes sense.

Anyway, by applying a torque instead of a raw force, I had to scale down the torsion constant (`C_AHO` in `util.h`) down to `1E-18 Nm.rad-1`. The bond angle potential is now correctly computed.

But something is now heavily getting on my nerves. What I thought was fixed isn't. The LJ potential is fucky.
